src/ncurses_ui.cc

```diff
@@ -1061,9 +1082,31 @@ void NCursesUI::set_ui_options(const Options& options)
     }
 
     {
-        auto it = options.find("ncurses_status_on_top"_sv);
-        m_status_on_top = it != options.end() and
-            (it->value == "yes" or it->value == "true");
+        auto it = options.find("ncurses_layout"_sv);
+        if (it != options.end())
+        {
+            auto layout = it->value | split<String>(',') | gather<Vector<String>>();
+            auto main = find(layout, "main");
+            auto status = find(layout, "status");
+            if (main == m_layout.end() or status == m_layout.end())
+                write_to_debug_buffer("main and status zones are mandatory in ncurses_layout");
+
+            m_layout = layout;
+            m_status_on_top = status < main;
```

Comment:
Do we still need m_status_on_top ? It looks like we should be able to get rid of it.